src/clings.h

```diff
@@ -2,6 +2,6 @@
 #ifndef CLINGS_CLINGS_H
 #define CLINGS_CLINGS_H
 
-int clings();
+int clings(void);
```

Comment:
why add void ?

Reply:
The flag `-Werror` also embeds the flag `-Wstrict-prototypes` which come with a sweet compilation message `error: a function declaration without a prototype is deprecated in all versions of C  [-Werror,-Wstrict-prototypes]`.

So for now the prototypes have to have a void parameter for no parameters.